lib/Service/DownloadModelsService.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace OCA\Recognize\Service;
+
+use OCA\Recognize\Helper\TAR;
+
+class DownloadModelsService {
+
+    /**
+     * @return void
+     * @throws \Exception
+     */
+    public function download() : void {
+        $targetPath = __DIR__ . '/../../models';
+        if (file_exists($targetPath)) {
```

Comment:
Is there some cleanup of old models happening somewhere else? Otherwise I guess new archives would never be downloaded with this.

Reply:
Ah, you're right. Shall we always force a new download?